docs/HowToCreateAWSSagemakerNotebooks.md

```diff
@@ -1,6 +1,6 @@
 # Using SageMaker Notebooks
 
-AWS offers different types of Notebook instances: SageMaker Studio, SageMaker Notebook Instances, and SageMaker Studio Lab. SageMaker Notebook Instances offer the most flexibility in terms of installing local software via conda/mamba or launching from custom containers. [SageMaker Studio](https://docs.aws.amazon.com/sagemaker/latest/dg/studio.html) and [SageMaker Studio Lab](https://aws.amazon.com/sagemaker/studio-lab/) allow for 'on the fly' machine resizing and notebook scheduling, as well as not worrying about resource availability, but they run in a managed environment and offer less flexibility for installing custom software. Most machine-learning related software are pre-installed, but these can be hard to use for a lot of bioinformatic tasks where you need to install CLI tools with conda.
+AWS offers different types of Notebook instances: SageMaker Studio, SageMaker Notebook Instances, and SageMaker Studio Lab. SageMaker Notebook Instances offer the most flexibility in terms of installing local software via conda/mamba or launching from custom containers. [Amazon SageMaker Notebook Instance](https://docs.aws.amazon.com/sagemaker/latest/dg/nbi.html) allows for 'on the fly' machine resizing and notebook scheduling, as well as not worrying about resource availability, but they run in a managed environment and offer less flexibility for installing custom software. Most machine-learning related software are pre-installed, but these can be hard to use for a lot of bioinformatic tasks where you need to install CLI tools with conda.

```

Comment:
Hey Hadi,

Here are some suggested adjustments to the paragraph here.

AWS offers various types of notebook instances: SageMaker Studio, SageMaker Notebook Instances, and SageMaker Studio Lab. SageMaker Notebook Instances provide flexibility when installing local software via conda or mamba and for using custom containers. The [Amazon SageMaker Notebook Instance](https://docs.aws.amazon.com/sagemaker/latest/dg/nbi.html) supports on-the-fly machine resizing and notebook scheduling, and handles resource availability concerns. However, it operates in a managed environment, offering less flexibility for installing custom software. While most machine-learning-related software is pre-installed, this setup can be challenging for bioinformatics tasks that require installing CLI tools with conda.